TYPO3.TYPO3CR/Tests/Behavior/Features/Basic/MoveNode.feature

```diff
@@ -28,6 +28,22 @@ Feature: Move node
       | user-admin |
     Then I should have 0 nodes
 
+  @fixtures
+  Scenario: Move an unpublished node (into) in user workspace and get by identifier
```

Comment:
Note: Since the bug is nondeterministic this does not always fail without the fix.